lib/util/index.ts

```diff
@@ -0,0 +1,270 @@
+import * as errors from 'balena-errors';
+import assign = require('lodash/assign');
+import cloneDeep = require('lodash/cloneDeep');
+import isArray = require('lodash/isArray');
+import isFunction = require('lodash/isFunction');
+import isNumber = require('lodash/isNumber');
+import isString = require('lodash/isString');
+import throttle = require('lodash/throttle');
+import * as memoizee from 'memoizee';
+import * as moment from 'moment';
+import BalenaSdk = require('../../typings/balena-sdk');
+import { getOsUpdateHelper as updateHelper } from './device-actions/os-update';
+import * as dt from './device-types';
+
+export const getOsUpdateHelper = updateHelper;
+export const deviceTypes = dt;
+
+export const notImplemented = () => {
+	throw new Error('The method is not implemented.');
+};
+
+export const onlyIf = (condition: boolean) => (fn: () => void) => {
```

Comment:
You can make this
```suggestion
export const onlyIf = (condition: boolean) => {
	if (condition) {
		return _.identity
	} else {
		return _.constant(notImplemented)
	}
}
```
so that the condition is only checked once

Reply:
That's clever :) But tbh I would prefer to keep the lodash imports to the minimum, so that we can eventually drop it.

Reply:
Well the equivalent plain js would be
```suggestion
export const onlyIf = (condition: boolean) => {
	if (condition) {
		return (x) => x
	} else {
		return() => notImplemented
	}
}